cterasdk/core/plans.py

```diff
@@ -9,10 +12,21 @@ class Plans(BaseCommand):
     """
     default = ['name']
 
-    def get(self, name, include=None):
+    def get(self, name):
         """
         Get a subscription plan
 
+        :param str name: Name of the subscription plan
+        """
+        try:
+            return self._portal.get('/plans/' + name)
+        except CTERAException as error:
+            raise CTERAException('Could not find subscription plan', error)
+
+    def get_multi(self, name, include=None):
```

Comment:
What makes this API multi?

Reply:
Fixed

Reply:
On the Portal, you can request the entire object (using 'GET') or ask for multiple attributes of the object using the get_multi command. I don't like retrieving the full object unless it's necessary.